tailwind.config.js

```diff
@@ -81,7 +81,7 @@ module.exports = {
       },
       fontFamily: {
         sans: ["Poppins", "sans-serif"],
-        brand: ["'Baloo Da 2'", "sans-serif"],
+        brand: ["The Bold Font"],
```

Comment:
In case the font can't load, a fallback would be good:
```suggestion
        brand: ["The Bold Font", "sans-serif"],
```